regression/pageObjects/productCategories.js

```diff
@@ -188,10 +188,14 @@ class productCategories {
         let NFTs = element.all(by.id("f52e1fdb-cca2-46db-a19b-8e7a446f64d6"));
         await NFTs.click();
     };
-    async clickOnSculpture(){
-        let sculpture = element.all(by.xpath("//h1[normalize-space()='Sculpture']"));
-        await sculpture.click();
-    };
+    async clickOn(ids){
+        let el = element.all(by.id(ids));
+        await el.click();
+    }
+    async clickOnX(xpa){
```

Comment:
could you use full word, xpa is not intuitive, I believe you it should represent xpath, but you should use something like clickOnFilters(filter) so that someone else looking at your code could easily understand and perhaps continue working on it